scripts/presubmit-tests.sh

```diff
@@ -317,8 +317,14 @@ function main() {
   fi
 
```

Comment:
Also please update the README.md with the new feature.

Reply:
Updated the README to include the new configuration.